### DISCUSSION TITLE
Laggiest with graphs

### DISCUSSION THREAD

**Discussion by potofpie** [elrumordelaluz/reactour#337] [Q&A]
I am using `reactour` on a page with 70+ `Highchart` graphs. When starting the tour there is 15 to 30 seconds of before the tour starts. I have tried to add a loading bar for when this change occurs but then the loading bar freezes while the tour starts. Does anyone have any suggestion to optimize this start process or account for the lag in a more elegant way?

**Comment by elrumordelaluz** [comment]
@potofpie mind sharing a small reproduction in a sandbox?
